Makefile

```diff
@@ -145,39 +156,59 @@ run-% : $(OBJDIR)/%
 	@tests/tap.sh $(<:build/%=%)
 
 test : $(tests:$(OBJDIR)/%=run-%) $(examples:$(OBJDIR)/%=run-%)
+# run test target in parallel
+tst : ;@$(MAKE) $(MFLAGS) V=$(V) test
+# CPU C tests only for backend %
+ctc-% : $(ctests);@$(foreach tst,$(ctests),$(tst) /cpu/$*;)
 
 prove : $(tests) $(examples)
 	$(PROVE) $(PROVE_OPTS) --exec 'tests/tap.sh' $(tests:$(OBJDIR)/%=%) $(examples:$(OBJDIR)/%=%)
+# run prove target in parallel
+prv : ;@$(MAKE) $(MFLAGS) V=$(V) prove
 
 examples : $(examples)
 
 $(ceed.pc) : pkgconfig-prefix = $(abspath .)
 $(OBJDIR)/ceed.pc : pkgconfig-prefix = $(prefix)
-.INTERMEDIATE: $(OBJDIR)/ceed.pc
+.INTERMEDIATE : $(OBJDIR)/ceed.pc
 %/ceed.pc : ceed.pc.template | $$(@D)/.DIR
 	@sed "s:%prefix%:$(pkgconfig-prefix):" $< > $@
 
+# OCCA env, OKL files cache, clear & rule to install
+OCCA_CACHE_DIR     ?= ${HOME}/.occa
+OCCA_LIB_CACHE_DIR := $(OCCA_CACHE_DIR)/libraries
+OKL_KERNELS        := $(shell find backends/occa -type f -name '*.okl')
+OKL_CACHED_KERNELS := $(subst backends/occa,$(OCCA_LIB_CACHE_DIR)/ceed,$(OKL_KERNELS))
```

Comment:
These paths don't exist, even after installation. It appears to depend on a hash that we probably don't know in advance. I'm going to streamline this to just cache `OKL_KERNELS` using the phony target.